css/style.css

```diff
@@ -42,42 +41,176 @@ Developer @GA in NYC
-      background-size: cover;
+  font-family: "Open Sans", sans-serif;
+  background-color: #f0efef;
 }
 
+/* Sets width of inner sections */
+.content-width {
+  width: 60%;
+  margin: 0 auto;
+}
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+h1 {
+  margin: 0;
+  padding: 40px 0 0 0;
+  font-size: 3em;
+  font-weight: 800;
+  color: rgb(255, 255, 255);
+}
 
-/* Center the container */
-.container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+h2 {
+  text-align: center;
+  font-size: 3em;
+  color: rgb(255, 255, 255);
+  font-weight: 600;
+  margin: 100px 0;
 }
 
+h3 {
+  font-weight: 600;
+  color: #033048;
+  font-size: 2.5em;
+}
+
+/* Moves items to the left outside container */
+.shift-left {
+  position: relative;
+  left: -35px;
+}
+
+/******************************************
+ HEADER STYLES
+*******************************************/
+
 header {
+  background: url("../images/header_bg.jpg") no-repeat center top;
+  background-size: cover;
+  height: 900px;
+  width: 100%;
+}
 
+header a {
+  height: 70px;
+  width: 250px;
+  text-align: center;
+  display: block;
+  background-color: #f9e42e;
+  padding: 15px 0 0 0;
+  text-decoration: none;
+  font-weight: 800;
+  color: #033048;
+  font-size: 1.7em;
+  border-radius: 5px;
+  margin: 0 auto;
 }
 
-footer {
+/******************************************
+ MAIN STYLES
+*******************************************/
+
+main ul,
+main p {
+  font-size: 1.5em;
+}
 
+main section:first-child {
+  padding: 0 0 70px 0;
+}
+
+main p {
+  width: 80%;
+}
+
+main ul {
+  width: 50%;
+}
+
+main ul {
+  position: relative;
+  left: 60px;
+  margin: 40px 0 0 0;
+}
+
+#quote-area {
+  background: url("../images/testimonial_bg.jpg") no-repeat top left;
+  background-size: cover;
+  width: 100%;
+  height: 400px;
+}
+
+#quote-area section {
+  width: 670px;
+  text-align: center;
+}
+
+blockquote {
+  font-weight: 300;
+  color: hsl(0, 0%, 100%);
+  font-size: 2.3em;
+  font-style: italic;
+  padding: 50px 0 0 0;
+  margin: 0;
+}
+
+#quote-area h3 {
+  font-size: 2.3em;
+  color: hsl(0, 0%, 100%);
+  margin: 0;
+}
+
+#sign-up {
+  background: #f9e42e;
+  height: 150px;
+  text-align: center;
+  padding: 40px 0;
+}
+
+#sign-up a {
+  color: hsl(0, 0%, 100%);
+  text-decoration: none;
+  font-weight: 800;
+  font-size: 1.7em;
+  border-radius: 5px;
+  background: #033048;
+  height: 70px;
+  width: 350px;
+  display: block;
+  margin: auto;
+  padding: 15px 0 0 0;
 }
 
 /******************************************
-/* ADDITIONAL STYLES
-/*******************************************/
+FOOTER STYLING
+*******************************************/
+
+footer {
+  background: #121212;
+  width: 100%;
+  color: rgb(255, 255, 255);
+  text-align: center;
+  margin: 0;
+}
 
+footer h1 {
+  color: #f9e42e;
+  padding: 50px 0;
+}
 
+footer ul li {
+  list-style: none;
+  display: inline;
+}
 
+footer li {
```

Comment:
:hammer: The images in the list are _almost_ centered. You're so close. I would use Chrome's inspector to analyze this and figure out how you can center these icons just right. Try hovering over the `<ul>` and the `<li>`'s. Can you see the margins and padding? What adjustments to these could you make to get it just right?